shell/ci/release/release.sh

```diff
@@ -27,6 +27,14 @@ send_failure_notification() {
 # shellcheck source=../../lib/logging.sh
 source "${LIB_DIR}/logging.sh"
 
+# shellcheck source=../../lib/bootstrap.sh
+source "${LIB_DIR}/../../lib/bootstrap.sh"
+
+if [[ "$(is_service)" == "false" ]]; then
```

Comment:
This is going to make it so that all CLIs require opslevel. I'd imagine we want to have a config setting in the box config to allow enabling and disabling this, right? Otherwise OSS would require OpsLevel now

Reply:
Yeah... This creates an annoying situation once again. We would ideally turn this off by default and have all internal repos enable this. In the current state, however, there is no easy way for us to enable it for all internal repos.

I am open to ideas here. I wouldn't want to make OSS opt out either though.

Reply:
We can turn it off by default in the scripts, that's fine. We can make it a global default for _our_ org, that's the main purpose of the box config.

Reply:
Oh, I see. For some reason I was stuck on the idea of turning this off at the stencil level.